protx/utils/maltreatment.py

```diff
@@ -102,13 +110,96 @@ def maltrt_stacked_bar(maltrt_data_dict):
 def maltreatment_plot_figure(area, geoid, variables, unit):
     logger.info("Selected maltreatment variables are: {}".format(variables))
     user_select_data = {
-        'area': area,
-        'geoid': geoid,
-        'units': unit,
-        'variables': ','.join(['"{}"'.format(v) for v in variables])
+        "area": area,
+        "geoid": geoid,
+        "units": unit,
+        "variables": ",".join(['"{}"'.format(v) for v in variables]),
     }
     db_conn = sqlite3.connect(db.cooks_db)
     maltrt_data = query_return(user_select_data, db_conn)
     db_conn.close()
     plot_figure = maltrt_stacked_bar(maltrt_data)
     return json.loads(plot_figure.to_json())
+
+
+def get_child_mal_by_age_charts(area, geoid):
+    db_conn = sqlite3.connect(db.cooks_db)
+    light_green_to_blue_color_palette = [
```

Comment:
it would be best to import and use the existing `light_green_to_blue_color_palette` in `protx/conf/styles.py`

Reply:
Ah overlooked that. Updated!